src/scripts/configure.sh

```diff
@@ -7,7 +7,7 @@ fi
 
 ORB_ENV_ACCESS_KEY_ID=$(circleci env subst "\$$ORB_ENV_ACCESS_KEY_ID")
 ORB_ENV_SECRET_ACCESS_KEY=$(circleci env subst "\$$ORB_ENV_SECRET_ACCESS_KEY")
-ORB_EVAL_AWS_CLI_REGION=$(circleci env subst "\$$ORB_EVAL_AWS_CLI_REGION")
+ORB_EVAL_AWS_CLI_REGION=$(circleci env subst "$ORB_EVAL_AWS_CLI_REGION")
```

Comment:
If this is an `EVAL` variable, shouldn't we use `eval echo` instead of `circleci env subst`?

Since it's not a directory, we can make it an "STR" instead.

Reply:
`PR`   #141  contains all the renaming of variables. Once that's merged, I'll deal with the conflicts and make the changes here.